src/ansys_sphinx_theme/__init__.py

```diff
@@ -463,6 +378,138 @@ def configure_theme_logo(app: Sphinx):
         theme_options["logo"] = logo_option
 
 
+def convert_pdf_to_png(pdf_path: pathlib.Path, output_dir: pathlib.Path, output_png: str):
+    """
+    Convert PDF to PNG images.
+
+    Parameters
+    ----------
+    pdf_path : pathlib.Path
+        Path to the PDF file.
+    output_dir : pathlib.Path
+        Path to the output directory.
+    output_png : str
+        Name of the output PNG file.
+    """
+    try:
+        doc = pymupdf.open(pdf_path)  # open document
+        for page in doc:
+            pix = page.get_pixmap()
+            pix.save(os.path.join(output_dir, output_png))
```

Comment:
Please, avoid using `os.path`. Use `pathlib` instead:

```suggestion
            pix.save(output_dir / output_png)
```